### PR TITLE
Playwright quickstart guide

### PR DESCRIPTION
**Pull request by spider-sauce**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/web-apps/automated-testing/playwright.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+## What You'll Need
+
+* A [Sauce Labs](https://saucelabs.com/) account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
+* [Download and install `saucectl`](/testrunner-toolkit/installation#installing-testrunner-toolkit)
+    * Ensure you've met the prerequisites outlined in the [Testrunner Toolkit documentation](/testrunner-toolkit/installation#what-youll-need)
+
+## Configure `saucectl`
+
+Before you begin running playwright tests, you must configure `saucectl` if you plan to view test results on the Sauce Labs dashboard or running tests on the Sauce Labs VMs:
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/playwright.md:19]: 💡 We actually require an account no matter which mode you are running now 💡

**Review by alexplischke — APPROVED** [review]
One small comment. LGTM otherwise!

File: docs/web-apps/automated-testing/playwright.md (hunk 1 above)
Comment (spider-sauce) [inline docs/web-apps/automated-testing/playwright.md:19]:
```suggestion
Before you begin running playwright tests, you must configure `saucectl`: 
```

Comment (spider-sauce) [reply docs/web-apps/automated-testing/playwright.md:19]: @alexplischke so like this ☝️

**Pull request merged**